### DISCUSSION TITLE
Add a comment without starting a review

### DISCUSSION THREAD

**Discussion by geoff-va** [pwntester/octo.nvim#552] [Q&A]
Is it possible to add a comment without starting a review? I can only seem to add comments after starting a review. Unless I'm just missing the correct context to do this in?

**Comment by williambdean** [comment]
It is not possible. And it is not possible in the GitHub UI either

**Reply by geoff-va** [reply]
I've frequently used the "Add single comment" button when I wanted to provide quick feedback vs starting a review and submitting it in the past.

This is possible in the github web interface?

![image](https://github.com/user-attachments/assets/cb86187f-f485-434f-83f5-881beea8123a)

**Reply by williambdean** [reply]
Ah, sorry. Yeah, you are right. 
I am not sure what is happening under the hood there. Would you like to do some investigation? Maybe a review is actually started, comment added, then submitting

Would you be interested in making a PR?

**Reply by williambdean** [reply]
Maybe this: https://docs.github.com/en/graphql/reference/mutations#addpullrequestreviewcomment ?

Edit: some of the docs say that is deprecating

**Reply by geoff-va** [reply]
Or maybe more generically this one? Seems to apply to "Issue or Pull Request":

https://docs.github.com/en/graphql/reference/mutations#addcomment
